common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1584,6 +1593,24 @@ def can_toggle_course_pacing(self):
             return False
         return datetime.now(utc) <= self.start
 
+    def set_discussion_toggle(self, value):
+        # Child of a course would be sections/chapters
+        for child in self.get_children():
+            child.set_discussion_toggle(value)
```

Comment:
As discussed above, you should just remove this, it's not as helpful as I thought it would be.